Dockerfile.go_modules

```diff
@@ -0,0 +1,195 @@
+FROM ubuntu:20.04
+
+ARG TARGETARCH=amd64
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+### SYSTEM DEPENDENCIES
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+  LC_ALL="en_US.UTF-8" \
+  LANG="en_US.UTF-8"
+
+RUN apt-get update \
+  && apt-get upgrade -y \
+  && apt-get install -y --no-install-recommends \
+    build-essential \
+    dirmngr \
+    git \
+    git-lfs \
+    bzr \
+    mercurial \
+    gnupg2 \
+    ca-certificates \
+    curl \
+    file \
+    zlib1g-dev \
+    liblzma-dev \
+    libyaml-dev \
+    libgdbm-dev \
+    bison \
+    tzdata \
+    zip \
+    unzip \
+    locales \
+    openssh-client \
+    software-properties-common \
+    # Everything from here onwards is only installed to ensure
+    # Python support works with all packages (which may require
+    # specific libraries at install time).
```

Comment:
why not drop all the python libs?

or rather, why not start from a fresh / empty build and then add packages as needed?